lib/ansible/modules/network/fortios/fortios_ipv4_policy.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/python
+#
+# Ansible module to manage IPV4 policy objects in fortigate devices
+# (c) 2016, Benjamin Jolivot <bjolivot@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {
+    'status': ['preview'],
+    'supported_by': 'community',
+    'version': '1.0'
+}
+
+DOCUMENTATION = """
+---
+module: fortios_ipv4_policy
+version_added: "2.3"
+author: "Benjamin Jolivot (@bjolivot)"
+short_description: Manage fortios firewall ipv4 policy objects
+description:
+  - This module provides management of firewall ipv4 policies on FortiOS devices.
+extends_documentation_fragment: fortios
+options:
+  id:
+    description:
+      - Policy ID.
+    required: true
+  state:
+    description:
+      - Specifies if address need to be added or deleted.
+    required: true
```

Comment:
according to arg spec the default is `present` and as it has a default, it is not required.